views/dashboard.handlebars

```diff
@@ -15,10 +15,71 @@
       {{/each}}
   </ul>
 </div>
-</div>
-</div>
 
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
+body {
+  font-family: "Lato", sans-serif;
+}
+
+.sidenav {
+  width: 180px;
+  position: fixed;
+  z-index: 1;
+  top: 20px;
+  left: 10px;
+  background: #eee;
+  overflow-x: hidden;
+  padding: 8px 0;
+  font-family:"Lato";
+  text-align: center;
+}
+
+.sidenav a {
+  text-decoration: none;
+  font-size: 25px;
+  color: #2196F3;
+  display: block;
+}
 
+.sidenav a:hover {
+  color: #064579;
+}
+
+.main {
+  margin-left: 140px; /* Same width as the sidebar + left position in px */
+  font-size: 28px; /* Increased text to enable scrolling */
+  padding: 0px 10px;
+}
+
+@media screen and (max-height: 450px) {
+  .sidenav {padding-top: 15px;}
+  .sidenav a {font-size: 18px;}
+}
+</style>
+</head>
+<body>
```

Comment:
Lines 20-64 cannot be in this file, css needs to be handled in our css file and the opening html elements are already in our main.handlebars file.